packages/wix-storybook-utils/src/AutoTestkit/unidriver/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { API } from './components/api-table';
+import { CodeExample } from './components/code-example';
+
+const driverNames = ['react', 'enzyme', 'puppeteer', 'protractor'];
+
+export const UnidriverTestkit = ({ component }) => (
+  <div>
+    {component.drivers.map(({ file, descriptor, error }, i) => {
+      if (error) {
+        return <span />;
```

Comment:
```suggestion
        return null;
```
what's the span for? affects layout maybe?